index.js

```diff
@@ -3,5 +3,5 @@
 const { Claim } = require('my-lib-54357435');
 const claim = new Claim('tenantId', '123');
 
-/** @type {AuthenticatedRequest} */ // TODO: be able to use my-lib.http.AuthenticatedRequest
+/** @type {import('my-lib-54357435').AuthenticatedRequest} */
 const request = {};
```

Comment:
I'm getting an "invalid type syntax" error here
![image](https://user-images.githubusercontent.com/287480/51874672-35ecf680-2362-11e9-8e83-1a7f64d9d89e.png)

Reply:
Seems to be fine in VS Code and also has some great IntelliSense support here, so please ignore this.